docs/import/manage_disaster.js

```diff
@@ -512,6 +530,104 @@ function createAssetDropdown(
+    event, propertyPath, expectedColumns, type, state) {
+  const newAsset = $(event.target).val();
+  handleAssetDataChange(newAsset, propertyPath);
+  return verifyAsset(newAsset, type, state, expectedColumns);
+}
+
+// Map of asset picker (represented by a string '<type>-<state>' e.g.
+// 'poverty-WA') to the most recent value of the picker. This helps us ensure
+// that we're only updating the status box for the most recently selected value.
+const lastSelectedAsset = new Map();
+
+/**
+ * Verifies an asset exists and has the expected columns.
+ * @param {string} asset
+ * @param {string} type values from the first index of each entry in {@code
+ *     scoreAssetTypes}
+ * @param {string} state e.g. 'WA'
+ * @param {Array<string>} expectedColumns
+ * @return {Promise<void>} returns null if column checking wasn't needed.
+ * Otherwise returns a promise that resolves when column checking is finished
+ * and select border color is updated.
+ */
+function verifyAsset(asset, type, state, expectedColumns) {
+  // TODO: disable kick off button until all green?
+  const tdId = type + '-' + state;
+  const select = $('#select-' + assetSelectionRowPrefix + type + '-' + state);
+  lastSelectedAsset.set(tdId, asset);
+  const assetMissingErrorFunction = (err) => {
+    if (err.includes('\'' + asset + '\' not found.')) {
+      updateColorAndHover(select, 'red', 'Error! asset could not be found.');
+    } else {
+      console.error(err);
+      updateColorAndHover(select, 'red', 'Unknown error.');
+    }
+  };
+  if (asset === '') {
+    updateColorAndHover(select, 'white', '');
+  } else if (expectedColumns.length === 0) {
+    updateColorAndHover(select, 'yellow', 'Checking columns...');
+    // TODO: is there a better way to evaluate feature collection existence?
+    convertEeObjectToPromise(ee.FeatureCollection(asset).first())
+        .then(() => updateColorAndHover(select, 'green', 'No expected columns'))
+        .catch(assetMissingErrorFunction);
+  } else {
+    updateColorAndHover(select, 'yellow', 'Checking columns...');
+    return convertEeObjectToPromise(getColumnsStatus(asset, expectedColumns))
+        .then((error) => {
+          if (lastSelectedAsset.get(tdId) === asset) {
+            if (error) {
+              updateColorAndHover(
+                  select, 'red',
+                  'Error! asset does not have all expected columns: ' +
+                      expectedColumns);
+            } else {
+              updateColorAndHover(
+                  select, 'green', 'Success! asset has all expected columns');
+            }
+          }
+        })
+        .catch(assetMissingErrorFunction);
+  }
+}
+
+/**
+ * Updates the border of the select and the hover text of the span.
+ * @param {JQuery<HTMLSelectElement>} select
+ * @param {string} color
+ * @param {string} title
+ */
+function updateColorAndHover(select, color, title) {
+  select.prop('style', 'border:2px solid ' + color).prop('title', title);
```

Comment:
nit: can you just specify `border: <color>` here and set `border: 2px solid` elsewhere (like, even, in a manage_disaster.css class)? Don't actually know if CSS works that way